ktor-server/ktor-server-tests/jvmAndNix/test/io/ktor/tests/server/plugins/CookiesTest.kt

```diff
@@ -377,3 +383,49 @@ class ParserServerSetCookieTest {
         assertEquals("/", parsed.path)
     }
 }
+
+class SecureCookiesTest {
```

Comment:
This test should be in `jvm` folder, because Ktor for `native` doesn't support session

Reply:
moved to `jvm` folder